recipes/lbapcommon/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "LbAPCommon" %}
+{% set version = "0.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/LbAPCommon-{{ version }}.tar.gz
+  sha256: 0713326ec22bdba7292c51dac23a3ebb4747a3bdddbe8f5f51898209df54d045
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python !=3.0,!=3.1,!=3.2,!=3.3,!=3.4,!=3.5
```

Comment:
Why not >=3.6?

Reply:
Unfortunately I still have to care about 2.7 😞